composer.json

```diff
@@ -39,8 +39,12 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpspec/phpspec": "^4.2 || ^5.0 || ^6.0",
-        "phpunit/php-code-coverage": "^5.0 || ^6.0 || ^7.0"
+        "phpspec/phpspec": "^5.0 || ^6.0",
+        "phpunit/php-code-coverage": "^6.0 || ^7.0"
+    },
+    "conflict": {
+        "friendsofphp/php-cs-fixer": "2.16.0",
```

Comment:
I had to add this because of failing low-deps tests related to this commit on PHP 7.4:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/121e0a6135e654466fcd1163a78b423c4fffee55